great_expectations/data_context/util.py

```diff
@@ -277,6 +278,24 @@ def parse_substitution_variable(substitution_variable: str) -> Optional[str]:
         return None
 
 
+def default_checkpoints_exist(directory_path: str) -> bool:
+    checkpoints_directory_path: str = os.path.join(
+        directory_path,
+        DataContextConfigDefaults.DEFAULT_CHECKPOINT_STORE_BASE_DIRECTORY_RELATIVE_NAME.value,
+    )
+    return (
+        os.path.isdir(checkpoints_directory_path)
+        and len(
+            [
+                os.path.join(checkpoints_directory_path, filename)
+                for filename in os.listdir(checkpoints_directory_path)
+                if filename.endswith(r".yml")
+            ]
+        )
+        > 0
+    )
+
+
```

Comment:
This looks great, but don't see it called anywhere. Since this looks like what you had in your upgrade helper, shouldn't the code there also be replaced with a call to this function?

Reply:
Fixed.